src/tests/unit/tests/electron/views/automated-checks/__snapshots__/automated-checks-view.test.tsx.snap

```diff
@@ -1,5 +1,40 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`AutomatedChecksView renders scanning spinner: automated checks view 1`] = `
+<React.Fragment>
+  <TitleBar
+    deps={
+      Object {
+        "currentWindow": null,
+        "deviceConnectActionCreator": null,
+        "scanActionCreator": proxy {
+          "___id": "BCDF5CE5-F0DF-40B7-8BA0-69DF395033C8",
```

Comment:
Potentially dumb question--where is this Guid coming from? Is it reproducible?

Reply:
from typemoq... it's reproducible... if not, the snapshot test will fail on each run... I'm not aware how do typemoq achieve that though.